src/app.py

```
@@ -1,10 +1,26 @@
-from flask import Flask, render_template
+from flask import Flask, render_template, jsonify
+from utils.route_tracker import get_current_position, load_route, haversine_distance
```

Ruff (0.8.2)

`utils.route_tracker.haversine_distance` imported but unused

Remove unused import: `utils.route_tracker.haversine_distance`

(F401)